app/assets/stylesheets/custom-bootstrap.css.scss

```diff
@@ -261,13 +272,17 @@ td form {
 .noborderbottom {
 	border:none;
 }
+
 .well-small {
 	min-height:10px;
 }
```

Comment:
It looks like parts of this file has the CSS indentation completely out of wack with other parts. If you could standardize it that would be great.